recipes-bsp/acpi-tables/acpi-tables.bb

```diff
@@ -20,6 +20,8 @@ B = "${WORKDIR}/acpi-tables"
 
 inherit deploy
 
+RDEPENDS_${PN} = "libgpiod"
```

Comment:
It's good to have for all of them, indeed.

Reply:
> It's good to have for all of them, indeed.

Except rdepends on libgpiod maybe. This causes libgpiod package to be installed when acpi-tables is installed. Maybe I should make rdepends conditional on MACHINE = edison?

Reply:
I think it's still good to have for all boards. But proceed as you consider better.